skrifa/src/color/traversal.rs

```diff
@@ -13,6 +13,33 @@ use super::{
     Brush, ColorPainter, ColorStop, PaintCachedColorGlyph, PaintError, Transform,
 };
 
+use alloc::vec::Vec;
+
+#[cfg(feature = "libm")]
+#[allow(unused_imports)]
+use core_maths::*;
+
+#[cfg(any(test, feature = "std"))]
+mod visited_set {
+    pub type VisitedSet = std::collections::HashSet<usize>;
+}
+
+#[cfg(not(any(test, feature = "std")))]
+mod visited_set {
```

Comment:
Please add a comment explaining why this implementation actually makes sense as at a glance it looks an awful lot like someone committed a wip by accident

Reply:
As opposed to committing a wip on purpose? :)